src/redux/services/validation.ts

```diff
@@ -177,7 +174,7 @@ function determineContainerIndex(
 ): YamlPath {
   for (let i = 0; i < properties.length; i += 1) {
     const property = properties[i];
-    const containers: {name: string}[] = get(resource.content, prefix.concat(property), []);
+    const containers = get<{name: string}[]>(resource.content, prefix.concat(property), []) ?? [];
```

Comment:
📓 TIL Lodash's get defaultValue only runs on `undefined` and not `null`.

In this case, some Helm charts incorrectly set `null` instead of `[]` to `initContainers` which causes the next line to crash because of accessing `findIndex` on `undefined`. It's their Helm charts not following spec but we should handle it gracefully.